bedrock/thunderbird/templates/thunderbird/includes/simple-footer.html

```diff
@@ -0,0 +1,22 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<footer id="colophon">
+  <div class="row">
+    <ul class="primary">
+      <li><a href="/thunderbird/features/"><span>{{_('Features')}}</span></a></li>
+      <li><a href="https://addons.mozilla.org/thunderbird/"><span>{{_('Add-ons')}}</span></a></li>
+      <li><a href="https://support.mozilla.org/products/thunderbird"><span>{{_('Support')}}</span></a></li>
```

Comment:
I think `rel="external"` is not necessary here because the linked sites are Mozilla properties. The Firefox landing page's footer doesn't have it neither.